lib/spack/docs/testing_guide.rst

```diff
@@ -0,0 +1,126 @@
+.. _testing-guide:
+
+=============
+Testing Guide
+=============
+
+This guide is intended for developers or administrators who want to
+integrate or test software releases built with Spack.
+
+---------
+Reporting
+---------
+The install and testing infrastructure of Spack supports multiple output formats for
+reporting the result of package builds in tools such as Jenkins or CDash.
+
+The output formats can be enabled by e.g.
+
+.. code-block:: console
+
+   $ spack install --log-format=junit <spec>
+
+Per default the log results will be placed into `var/<format>/test-<short_spec>.xml`.
+To change the default behaviour, use
+
+.. code-block:: console
+
+   $ spack install --log-file report --log-format=junit <spec>
+
+The logs contain one test case per package being built.
+
+
+
+----------------
+using test-suite
+----------------
+
+Test-suite is designed to read in a yaml file describing all packages and compilers along 
+with versions. Using the enabled field will allow you to focus on specific packages.
+To narrow down the scope of a package or compiler, you can use the exclusion field.
+
+--Example of a yaml file---
```

Comment:
Call this a "test suite file" or a "test suite YAML file" or something more specific.